images/image-awaiter/main.go

```diff
@@ -86,6 +85,8 @@ func main() {
 	flag.StringVar(&daemonSet, "daemonset", "hook-image-puller", "The name DaemonSet that will perform image pulling")
 	var debug bool
 	flag.BoolVar(&debug, "debug", false, "Communicate through a 'kubectl proxy --port 8080' setup instead.")
+	var strictCheckDuration int
```

Comment:
What do you think of a slightly more descriptive name, like 'schedule-wait-time'? with just `strict-check-duration` it's unclear what the check is for.

Reply:
I went with `pod-scheduling-wait-duration`